{"hierarchy":{"paths":[["doc:\/\/com.ibm.verify.ios.sdk\/documentation\/FIDO2","doc:\/\/com.ibm.verify.ios.sdk\/documentation\/FIDO2\/PublicKeyCredentialError"]]},"sections":[],"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/fido2\/publickeycredentialerror\/general(message:)"]}],"identifier":{"interfaceLanguage":"swift","url":"doc:\/\/com.ibm.verify.ios.sdk\/documentation\/FIDO2\/PublicKeyCredentialError\/general(message:)"},"abstract":[{"type":"text","text":"General error with custom message."}],"primaryContentSections":[{"kind":"declarations","declarations":[{"languages":["swift"],"tokens":[{"kind":"keyword","text":"case"},{"text":" ","kind":"text"},{"kind":"identifier","text":"general"},{"text":"(","kind":"text"},{"text":"message","kind":"externalParam"},{"kind":"text","text":": "},{"text":"String","preciseIdentifier":"s:SS","kind":"typeIdentifier"},{"text":")","kind":"text"}],"platforms":["iOS"]}]}],"kind":"symbol","schemaVersion":{"major":0,"minor":3,"patch":0},"metadata":{"roleHeading":"Case","role":"symbol","modules":[{"name":"FIDO2"}],"externalID":"s:5FIDO224PublicKeyCredentialErrorO7generalyACSS_tcACmF","title":"PublicKeyCredentialError.general(message:)","symbolKind":"case","fragments":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"text":"general","kind":"identifier"},{"kind":"text","text":"("},{"text":"message","kind":"externalParam"},{"kind":"text","text":": "},{"text":"String","kind":"typeIdentifier","preciseIdentifier":"s:SS"},{"kind":"text","text":")"}]},"references":{"doc://com.ibm.verify.ios.sdk/documentation/FIDO2/PublicKeyCredentialError":{"type":"topic","role":"symbol","url":"\/documentation\/fido2\/publickeycredentialerror","abstract":[{"text":"A type that indicates when Fido2ApiClient encounters an error.","type":"text"}],"identifier":"doc:\/\/com.ibm.verify.ios.sdk\/documentation\/FIDO2\/PublicKeyCredentialError","title":"PublicKeyCredentialError","kind":"symbol","fragments":[{"kind":"keyword","text":"enum"},{"text":" ","kind":"text"},{"text":"PublicKeyCredentialError","kind":"identifier"}],"navigatorTitle":[{"text":"PublicKeyCredentialError","kind":"identifier"}]},"doc://com.ibm.verify.ios.sdk/documentation/FIDO2/PublicKeyCredentialError/general(message:)":{"identifier":"doc:\/\/com.ibm.verify.ios.sdk\/documentation\/FIDO2\/PublicKeyCredentialError\/general(message:)","url":"\/documentation\/fido2\/publickeycredentialerror\/general(message:)","role":"symbol","title":"PublicKeyCredentialError.general(message:)","kind":"symbol","type":"topic","fragments":[{"text":"case","kind":"keyword"},{"kind":"text","text":" "},{"text":"general","kind":"identifier"},{"kind":"text","text":"("},{"text":"message","kind":"externalParam"},{"text":": ","kind":"text"},{"preciseIdentifier":"s:SS","text":"String","kind":"typeIdentifier"},{"text":")","kind":"text"}],"abstract":[{"text":"General error with custom message.","type":"text"}]},"doc://com.ibm.verify.ios.sdk/documentation/FIDO2":{"abstract":[],"identifier":"doc:\/\/com.ibm.verify.ios.sdk\/documentation\/FIDO2","url":"\/documentation\/fido2","title":"FIDO2","kind":"symbol","role":"collection","type":"topic"}}}